octobot_backtesting/importers/exchanges/exchange_importer.py

```diff
@@ -108,13 +108,42 @@ async def _init_available_data_types(self):
                                      if await self.database.check_table_exists(table)
                                      and await self.database.check_table_not_empty(table)]
 
```

Comment:
I don't understand the reason behind this method

Reply:
Before it we were all the time selecting the whole database regardless of required timestamps. Now using this in `get_ohlcv` etc, we can pass timestamps when selecting data to reduce the amount of selected data. This saves a lot of time when parsing it (json.loads takes a lot of time, it's speeding up the backtest a lot when running backtests on parts of a datafile to be able to select only needed data before parsing it all)

Reply:
OK, that's a great improvement 👍 thanks for the explanation